### PR TITLE
Add defer directive to the operations stats page

### PR DESCRIPTION
**Pull request by beerose**
### Checklist

- [ ] Input validation
- [ ] Output encoding
- [ ] Authentication management
- [ ] Session management
- [ ] Access control
- [ ] Cryptographic practices
- [ ] Error handling and logging
- [ ] Data protection
- [ ] Communication security
- [ ] System configuration
- [ ] Database security
- [ ] File management
- [ ] Memory management
- [ ] Testing

### CODE REVIEW

File: packages/web/app/src/lib/urql-cache.ts [hunk 1]
```diff
@@ -28,6 +29,14 @@ import {
   TokensDocument,
 } from '../graphql';
 
+export const getOperationName = (query: DocumentNode): string | undefined => {
+  for (const node of query.definitions) {
+    if (node.kind === Kind.OPERATION_DEFINITION) {
+      return node.name ? node.name.value : undefined;
```
Comment (B2o5T) [inline packages/web/app/src/lib/urql-cache.ts:35]:
```suggestion
      return node.name?.value;
```

File: package.json [hunk 2]
```diff
@@ -51,13 +51,13 @@
...
-    "@graphql-codegen/typescript": "3.0.4",
-    "@graphql-codegen/typescript-operations": "3.0.4",
-    "@graphql-codegen/typescript-resolvers": "3.2.1",
+    "@graphql-codegen/cli": "3.3.1-alpha-20230420202418-82620d460",
+    "@graphql-codegen/client-preset": "3.0.1-alpha-20230420202418-82620d460",
+    "@graphql-codegen/graphql-modules-preset": "3.1.3-alpha-20230420202418-82620d460",
+    "@graphql-codegen/typed-document-node": "4.0.1-alpha-20230420202418-82620d460",
+    "@graphql-codegen/typescript": "3.0.4-alpha-20230420202418-82620d460",
+    "@graphql-codegen/typescript-operations": "3.1.0-alpha-20230420202418-82620d460",
+    "@graphql-codegen/typescript-resolvers": "3.2.1-alpha-20230420202418-82620d460",
```
Comment (beerose) [inline package.json:60]: TODO: update after codegen is released

**Comment by kamilkisiela** [comment]
We could add `@defer` also here:
https://github.com/kamilkisiela/graphql-hive/blob/a7e623b3226003a5f944021622a420a34d280735/packages/web/app/src/graphql/fragments.graphql#L12

**Comment by beerose** [comment]
I'm going to close this PR as we decided not to use defer in Hive for now.

**Pull request closed without merging**